signer/threshold_validator.go

```diff
@@ -253,7 +253,7 @@ func (pv *ThresholdValidator) waitForPeerSetEphemeralSharesAndSign(
 	ourID int,
 	peer Cosigner,
 	hrst HRSTKey,
-	encryptedEphemeralSharesThresholdMap *map[Cosigner][]CosignerEphemeralSecretPart,
+	encryptedEphemeralSharesThresholdMap map[Cosigner][]CosignerEphemeralSecretPart,
```

Comment:
nit: It's rare to see long variable names in Go. `encryptedShares` is probably descriptive enough. You can quickly see the type for more info. I can't tell here, but `map[Cosigner][]CosignerEphemeralSecretPart` could be more readable as its own type or type alias.

Reply:
shrunk